### DISCUSSION TITLE
Will Passion Prevail Over Responsibility?

### DISCUSSION THREAD

**Discussion by aayushrg7** [harilvfs/carch#14] [Q&A]
Sometimes I wonder if you'll be able to keep working on the Carch project. Will you still have the time and passion for it with all the responsibilities and pressures you face? Will Carch continue to grow and improve in the coming days? I know it’s hard to keep up with a hobby or passion when life gets busy, and eventually, you might have to let it go.

**Comment by harilvfs** [comment]
You’re right; I’m not sure if I’ll be able to keep up with my passion for open-source, Linux, and my project, Carch. I didn’t think I’d face this level of responsibility so soon, and honestly, I wasn’t ready for it. With so much on my plate now, it seems like I might eventually have to let go.

But looking back, this journey has given me so much—knowledge, happiness, and a sense of purpose in exploring Linux and building Carch. It became a genuine source of joy for me, and even if I have to move on, this experience will always mean a lot.

Life has a way of making us let go of things we love, or sometimes they slip away. Right now, it’s tough to look back on those good times. But no matter what, the lessons, the memories, and the work I put into Carch will always stay with me. Open-source, Linux, and Carch will forever be a part of who I am.
